packages/kotti-ui/source/kotti-filters/KtFilters.vue

```diff
@@ -3,12 +3,14 @@
 		<FilterSearch
 			v-if="hasSearchColumn"
 			:column="searchColumn"
+			:dataTest="dataTest"
 			:filter="searchValue"
 			:isLoading="isLoading"
 			@input="setSearchFilter"
 		/>
 		<div ref="listTriggerRef">
 			<ButtonLink
+				:data-test="dataTest ? `${dataTest}-show-filters` : null"
```

Comment:
Should either be `:dataTest=` or `: undefined` depending on if this is intended to be passed as a prop or as a html attribute